config/application.rb

```diff
@@ -52,5 +52,7 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
 
     config.middleware.use PDFKit::Middleware, { print_media_type: true }, disposition: 'attachment', only: [%r[^/provider/applications/\d+]]
+
+    config.action_mailer.perform_deliveries = false if HostingEnvironment.sandbox_mode?
```

Comment:
This is a bit too broad I think, we're going to want emails for authentication, otherwise the candidate flow can't be tested. Similarly candidate notifications, referee emails, initial email to users to say that they've been invited are very useful (and I think wouldn't be too spammy, as they're triggered by actual user behaviour, not RDB/DBD robots).

Perhaps we can tag the providers notifications somehow, and add an interceptor to ignore those?

Reply:
That’s a very good point! Interceptor sounds like a good idea. I’ll have a look tomorrow.